packages/gopcaml-mode/gopcaml-mode.0.0.1/opam

```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+version: "0.0.1"
+synopsis:
+  "Ultimate Ocaml editing plugin, providing advanced structural editing, movement and analysis in Emacs"
+description:
+  "Gopcaml-mode is a plugin for emacs for editing OCaml code. It aims to extend existing Emacs editing experience to closer match the features in modern IDEs."
+maintainer: ["kirang@comp.nus.edu.sg"]
+authors: ["Kiran Gopinathan"]
+license: "GPL-3.0-only"
+homepage: "https://gitlab.com/gopiandcode/gopcaml-mode"
+bug-reports: "https://gitlab.com/gopiandcode/gopcaml-mode/issues"
+depends: [
+  "dune" {>= "2.1" & < "2.4"}
+  "core" {>= "v0.13.0" & < "v0.14.0"}
+  "ppx_deriving" {>= "4.4" & < "4.5"}
+  "ecaml" {>= "v0.13.0" & < "v0.14.0"}
+  "ocaml-compiler-libs" {>= "v0.12.1" & < "v0.12.2"}
```

Comment:
You could also link compiler-libs directly, the ones coming from the compiler itself, without the need for an extra dependency

Reply:
I couldn't find any packages called compiler libs on opam - how would I go about linking to the compiler directly?

Reply:
I set the project up a while back, so I can't recall the exact motivation, but I think I ended up importing ocaml-compiler-libs as I found that the compiler internals weren't exported to ocaml code without having an explicit dependency.

Reply:
I think using `compiler-libs.common` (or whatever other relevant lib) in the libraries section of the relevant dune files should work out of the box